packages/workers-shared/package.json

```diff
@@ -2,16 +2,50 @@
 	"name": "@cloudflare/workers-shared",
 	"version": "0.1.0",
 	"description": "Package that is used at Cloudflare to power some internal features of Cloudflare Workers.",
+	"keywords": [
+		"cloudflare",
+		"workers",
+		"cloudflare workers"
+	],
+	"homepage": "https://github.com/cloudflare/workers-sdk/tree/main/packages/workers-shared#readme",
+	"bugs": {
+		"url": "https://github.com/cloudflare/workers-sdk/issues"
+	},
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/cloudflare/workers-sdk.git",
+		"directory": "packages/workers-shared"
+	},
+	"license": "MIT OR Apache-2.0",
+	"author": "wrangler@cloudflare.com",
+	"files": [
+		"dist"
+	],
 	"scripts": {
-		"build": "pnpm run clean && pnpm run bundle:asset-server",
+		"build": "pnpm run clean && pnpm run bundle:asset-server:prod",
 		"bundle:asset-server": "esbuild asset-server-worker/src/index.ts --format=esm --bundle --outfile=dist/asset-server-worker.mjs --sourcemap=external",
+		"bundle:asset-server:prod": "pnpm run bundle:asset-server --minify",
+		"check:lint": "eslint . --max-warnings=0",
+		"check:type": "tsc",
 		"clean": "rimraf dist",
 		"dev": "pnpm run clean && concurrently -n bundle:asset-server -c blue \"pnpm run bundle:asset-server --watch\""
 	},
 	"devDependencies": {
+		"@cloudflare/eslint-config-worker": "workspace:*",
+		"@cloudflare/workers-tsconfig": "workspace:*",
+		"@cloudflare/workers-types": "^4.20240725.0",
```

Comment:
The latest version is now `4.20240806.0`

Reply:
fixed